hathor/manager.py

```diff
@@ -135,6 +138,7 @@ def __init__(self,
         self._settings = settings
         self.daa = daa
         self._cmd_path: Optional[str] = None
+        self._peer_id_json_path = peer_id_json_path
```

Comment:
Why not use sysctl and get a file path as argument? On the other hand, we could have other commands such as `refresh_ip_address()` or `refresh_public_ip_address()`.

Reply:
The downside is that the `peer.json` will not reflect the current state and that a restart would revert it to the original state.

Reply:
But I don't think the entrypoints are commonly stored there. I'd say, in most cases, it is autodetected by the auto hostname features in the full node. So it wouldn't revert to the original state.

Reply:
This was changed in https://github.com/HathorNetwork/hathor-core/pull/981/commits/83e2acafa848c1d610b95d23c3065dcf182f9bac

Reply:
Thanks!